tripleblindmarket/covid-safe-paths
# Feature/onboarding addtitional step

## Description

This PR builds off of the work done by @darkerthanblackoff in #619. The changes in this PR were performed in order to get the original PR pulled in to an upcoming release. 

The Google Maps onboarding feature that was added in #619 is not slated to be included for the release, so the additional commit on this PR removes that functionality, along with some minor UX enhancements (cut-off text in onboarding screens unrelated to this PR, scrollable authorities list).

#### Screenshots:

No Authorities Added Yet 
_Note:_ The "Choose health authority" button brings a user to the ChooseProvider screen
<img width="360" alt="Screen Shot 2020-04-23 at 8 50 20 PM" src="https://user-images.githubusercontent.com/20157849/80166655-0d26d180-85a4-11ea-9448-8f57afa51cb9.png">

Authorities Added (scrollable list)
<img width="355" alt="Screen Shot 2020-04-23 at 5 47 28 PM" src="https://user-images.githubusercontent.com/20157849/80165769-d2239e80-85a1-11ea-8857-e4cfdf43e0f2.png">

#### How to test

Walkthrough onboarding and:
 - Confirm that the header text on all screens is not cut off on the top
 - You are able to add a health authority, and after completing onboarding, the authority is in your subscribed list in settings
